lib/game_teams.rb

```diff
@@ -20,7 +22,7 @@ def self.win_percentage(grouped = @@all_game_teams_data.group_by(&:team_id))
     grouped = grouped
     grouped.map {|team, data| grouped[team] = data.map(&:result)}
     grouped.map do |team, data|
-      grouped[team] = (data.count("WIN")/data.size.to_f).round(2)
+      grouped[team] = (data.count("WIN")/data.size.to_f).round(5)
```

Comment:
I don't think I'm immediately understanding why we changed the rounding & whether that has any impact anywhere else. Let's just talk through it so we're all on same page.